mgmt/rest/v2/config.go

```diff
@@ -33,17 +33,47 @@ type PolicyTable cpolicy.RuleTable
 
 type PolicyTableSlice []cpolicy.RuleTable
 
-// cdata.ConfigDataNode implements it's own UnmarshalJSON
+// PluginConfigItem represents the response of a plugin config items.
+//
+// swagger:response PluginConfigResponse
 type PluginConfigItem struct {
-	cdata.ConfigDataNode
+	// in: body
+	Config cdata.ConfigDataNode `json:"config"`
```

Comment:
Exposed the config so that clients in different repositories/anywhere can reference it.